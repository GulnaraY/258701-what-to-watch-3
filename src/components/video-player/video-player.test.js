import React from 'react';
import rerender from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';
const src = `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`;
const poster = `poster`;

it(`Should render VideoPlayer`, () => {
  const tree = rerender.create(
      <VideoPlayer
        src={src}
        poster={poster}
        onVideoUnhover={()=>{}}
        isMute={true}
        isPlaying={true}
      />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
