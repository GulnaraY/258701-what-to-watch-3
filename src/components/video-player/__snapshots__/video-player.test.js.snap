// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render VideoPlayer 1`] = `
<video
  muted={true}
  poster="img/poster"
  width=""
>
  <source
    src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
    type="video/mp4"
  />
</video>
`;
